src/eva-icons/evaIcon.component.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { SvgProps } from 'react-native-svg';
+import { IconProvider } from 'react-native-ui-kitten';
+
+type IconElement = React.ReactElement<SvgProps>;
+type IconComponent = React.ComponentType<SvgProps>;
+
+export class EvaIcon implements IconProvider<SvgProps> {
+
+  constructor(private content: IconComponent) {
```

Comment:
I did not know, it is possible to initialize the class field in this way. Or I'm wrong?

Reply:
Yes, this is a TypeScript syntax sugar